test/sentry-cli-integration/server-output-Linux.expected

```diff
@@ -0,0 +1,586 @@
+     Sentry.Samples.Maui.dll.so
+     Xamarin.AndroidX.DrawerLayout.dll.so
+     System.Private.CoreLib.dll.so
+     libaot-System.Memory.dll.so
+     libaot-System.Numerics.Vectors.dll.so
+     libaot-System.Text.Encodings.Web.dll.so
+     Java.Interop.dll.so
+     libaot-Microsoft.Maui.dll.so
+     libaot-Microsoft.Extensions.Configuration.dll.so
+     System.Net.WebProxy.dll.so
+     Xamarin.AndroidX.Lifecycle.ViewModel.dll.so
+     Xamarin.AndroidX.DrawerLayout.dll.so
+     libaot-Sentry.Extensions.Logging.dll.so
+     libSystem.Security.Cryptography.Native.Android.so
+     System.Net.NetworkInformation.dll.so
+     System.Memory.dll.so
+     System.Collections.Specialized.dll.so
+     libaot-Sentry.Android.AssemblyReader.dll.so
+     libsentrysupplemental.so
+     libaot-Xamarin.Kotlin.StdLib.dll.so
+     Microsoft.Extensions.Primitives.dll.so
+     Xamarin.AndroidX.CursorAdapter.dll.so
+     Sentry.Android.AssemblyReader.dll.so
+     Sentry.Extensions.Logging.dll.so
+     libaot-Microsoft.Extensions.Options.dll.so
+     System.Diagnostics.DiagnosticSource.dll.so
+     libaot-System.Reflection.Metadata.dll.so
+     libaot-System.Linq.Expressions.dll.so
+     Xamarin.Google.Android.Material.dll.so
+     libaot-System.Private.Uri.dll.so
+     libaot-System.ComponentModel.dll.so
+     Xamarin.AndroidX.Collection.dll.so
+     libaot-Xamarin.AndroidX.DrawerLayout.dll.so
+     Xamarin.AndroidX.RecyclerView.dll.so
+     libaot-System.Runtime.InteropServices.RuntimeInformation.dll.so
+     System.dll.so
+     libaot-System.Security.Cryptography.Algorithms.dll.so
+     Microsoft.Maui.Controls.Compatibility.dll.so
+     Xamarin.AndroidX.CustomView.dll.so
+     libaot-System.Net.NetworkInformation.dll.so
+     libSystem.Security.Cryptography.Native.Android.so
+     Microsoft.Extensions.Options.ConfigurationExtensions.dll.so
+     System.Runtime.InteropServices.RuntimeInformation.dll.so
+     libxamarin-app.so
+     Xamarin.AndroidX.CoordinatorLayout.dll.so
+     libaot-Xamarin.AndroidX.AppCompat.dll.so
+     Xamarin.AndroidX.Lifecycle.LiveData.Core.dll.so
+     libaot-Xamarin.AndroidX.Fragment.dll.so
+     libaot-System.ComponentModel.dll.so
+     libaot-Microsoft.Maui.Graphics.dll.so
+     libaot-System.Threading.dll.so
+     libaot-Xamarin.AndroidX.CoordinatorLayout.dll.so
+     libaot-System.Runtime.dll.so
+     libaot-Xamarin.AndroidX.Activity.dll.so
+     System.ComponentModel.TypeConverter.dll.so
+     System.Numerics.Vectors.dll.so
+     libSystem.IO.Compression.Native.so
+     System.Xml.ReaderWriter.dll.so
+     libaot-Mono.Android.dll.so
+     System.Diagnostics.TraceSource.dll.so
+     libaot-Microsoft.Extensions.Logging.dll.so
+     libsentry-android.so
+     Microsoft.Maui.dll.so
+     libaot-System.Collections.NonGeneric.dll.so
+     Xamarin.AndroidX.Navigation.UI.dll.so
+     libaot-Xamarin.AndroidX.AppCompat.AppCompatResources.dll.so
+POST /api/0/projects/org/project/reprocessing/ HTTP/1.1
+GET /STOP HTTP/1.1
+HTTP server stopping!
```

Comment:
Before I finish this, I think we should have a look if we should really upload all those intermediary symbols. I know this came up before but I think it wasn't really resolved. Uploading 800+ files doesn't look like the greatest idea ever if we're not using them at all.

BTW it's possible to specify the `DifType` we want to upload when calling CLI `debug-files upload`:
```
    -t, --type <TYPE>                Only consider debug information files of the given type.  By
                                     default, all types are considered. [possible values:
                                     bcsymbolmap, dsym, elf, pe, pdb, portablepdb, sourcebundle,
                                     breakpad, proguard, wasm]
```

If we know we only need PPDB, let's only use that.

@mattjohnsonpint 
@bruno-garcia

Reply:
Agree on the tons of files, and yeah - we primarily need the `portablepdb` type.  Or the `pe` type if the symbols are embedded.  The problem is that we don't have a good sense of which debug files will be useful during a native crash on each platform.  If we were to be more selective, we'd need to specify different types and paths based on target platform and various other build properties.  We'd likely miss something.  Certainly it could be done, but would take a lot of experimentation.

Also, I think we'd need the ability to specify more than one type.  Not sure if that's already supported or not.

Reply:
catching everything form bin/Release should be fine for all the managed code (ppdb) but if we're building a mobile app, there'll be some native libraries that never make it to the bin/Release folder. So maybe we need different invocations. One for the .NET managed. And other(s) for the native stuff that only made somewhere into the `obj` folder

Reply:
BTW this is what's in the release folder
<img width="542" alt="image" src="https://user-images.githubusercontent.com/6349682/221676844-5d3a86fd-80d0-4685-b0d4-cf4ab902645f.png">

Reply:
Makes sense. That looks good.

If we only collect that folder, we'll miss out on debug files that were tripped off of native code that got into the `.apk`

Reply:
I've created a follow-up issue and disabled the test part that matches the expected file list temporarily. https://github.com/getsentry/sentry-dotnet/issues/2260